realm/src/androidTest/java/io/realm/NotificationsTest.java

```diff
@@ -173,10 +173,10 @@ public Boolean call() throws Exception {
         while (!isReady.get()) {
             Thread.sleep(5);
         }
-        Thread.sleep(100); 
+        Thread.sleep(100);
```

Comment:
you can use `SystemClock.sleep(millis)` if you don't want to handle `InterruptedException`